### PR TITLE
Fix bugs in UG

### LINKED ISSUE #187: [PE-D] UG Explanation
What is this index? In the orgininal list of in the recently viewed list? 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: youaremysky99/ped#8

### LINKED ISSUE #189: [PE-D] UG Typo
Should it be "remarks" for consistency? 

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: youaremysky99/ped#10

### LINKED ISSUE #191: [PE-D] UG Visualization
There are two examples but just one picture. How can I know which example this picture is refering to?

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: youaremysky99/ped#12

### LINKED ISSUE #199: [PE-D] Typo in the Command Features of User Guide
There should be an additional space between `n` and `/`

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: gloon99/ped#1

### PR DESCRIPTION
**Pull request by zhengweii**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -293,45 +364,75 @@ Example:
...
+  list of displayed suppliers in the GUI to `10` and gives `Panadol` a tag of `fever`.
+  
+![update warehouse product](images/updateWarehouseProduct.png)
+
+### Updating the quantity and/or tags of a product stored in a warehouse: `update`
+
+Updates the quantity and/or tags of the product with the specified name at the specified warehouse index.
+If the product does not exist, a new product will be created for that warehouse. 
+
+Format:	`update ct/w i/INDEX pd/PRODUCT_NAME [q/QUANTITY] [t/TAG]`
```
Comment (jeffreytjs) [inline docs/UserGuide.md:393]: Are the blank spaces meant to be?

**Review by jeffreytjs — COMMENTED** [review]
LGTM but the others should review it as well especially for those that has changes in the formatting to be implemented.

File: docs/UserGuide.md (hunk 1 above)
Comment (jeffreytjs) [reply docs/UserGuide.md:393]: I think is typo cos the rest don't have it

Comment (zhengweii) [reply docs/UserGuide.md:393]: Ahh I just checked! I think this is an issue with the rendering here! There are no extra blank spaces in the file!

**Review by tohyuting — APPROVED** [review]
LGTM

**Pull request merged**